src/lib/validator-select_multiple.ts

```diff
@@ -0,0 +1,54 @@
+import {ContactProperty} from '../config';
+import {IValidator} from './validation';
+import ValidatorString from './validator-string';
+import ValidatorSelectOne from './validator-select_one';
+
+export default class ValidatorSelectMultiple implements IValidator {
+  DELIMITER = ' ';
+
+  isValid(input: string, property: ContactProperty): boolean | string {
+    // Verify property.parameter is an object and is not null
+    if (!property?.parameter || typeof property.parameter !== 'object') {
+      throw new TypeError(`Expected attribute "parameter" on property ${property.property_name} to be an object.`);
+    }
+
+    const selectOneValidator = new ValidatorSelectOne();
+    const stringValidator = new ValidatorString();
+
+    const selectedValues = this.parseInput(input, stringValidator); 
+    const invalidValues = selectedValues.filter(
+      value => !selectOneValidator.isValid(value, property)
+    );
+
+    if (invalidValues.length > 0) {
+      return `Invalid values: ${invalidValues.join(', ')}`;
```

Comment:
Could we include the `property.friendly` in this?
```suggestion
      return `Invalid values for property "${property.friendly}": ${invalidValues.join(', ')}`;
```